src/build_index.rs

```diff
@@ -29,7 +29,7 @@ pub fn build_index<K: Kmer + Sync + Send>(
     // Thread pool Configuration for calling BOOMphf
     rayon::ThreadPoolBuilder::new()
         .num_threads(MAX_WORKER)
-        .build_global()?;
+        .build()?;
```

Comment:
If you were to call the build_index method multiple times, build_global would cause an error the second time.